internal/api/utils.go

```diff
@@ -267,6 +301,14 @@ func UpdateMemoryFromBytesToMiB(jsonData datatypes.JSON) map[string]interface{}
 						continue
 					}
 				}
+				cpu, ok := limits["cpu"].(map[string]interface{})
+				if ok {
+					err := convertCPU(cpu)
+					if err != nil {
+						fmt.Printf("error converting cpu in %s: %v\n", period, err)
+						continue
+					}
+				}
 			}
 
 			requests, ok := recommendationSection["requests"].(map[string]interface{})
```

Comment:
Whole `recommendationSection["requests"]`block is repetitive. 
[DRY]: could you extract this out in function and just pass recommendationSection["limits"]/recommendationSection["requests"]?

Reply:
DRY performed and comment added :+1: